CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This changelog records changes to stable releases since 1.50.2. "TBA" changes he
 - feat: simplify pretty print to align with devtools ([vscode#151410](https://github.com/microsoft/vscode/issues/151410))
 - fix: debugged child processes in ext host causing teardown ([#1289](https://github.com/microsoft/vscode-js-debug/issues/1289))
 - fix: errors thrown in process tree lookup not being visible ([vscode#150754](https://github.com/microsoft/vscode/issues/150754))
+- fix: extension debugging not working with two ipv6 interfaces ([vscode#144315](https://github.com/microsoft/vscode/issues/144315))
```

Comment:
I don't think it's actually caused directly by any interface, but by some logic deciding that because there's _some_ ipv6 interface, resolving `localhost` should resolve to ipv6 loopback instead of ipv4 loopback. This could, in theory, be a problem even without an ipv6 interface attached. Here though I just copied the title of the original interface.